account_invoice_report_grouped_by_picking/models/account_move.py

```diff
@@ -113,17 +113,19 @@ def lines_grouped_by_picking(self):
                 remaining_qty -= qty
             # To avoid to print duplicate lines because the invoice is a refund
             # without returned goods to refund.
-            if self.move_type == "out_refund" and not has_returned_qty:
+            if (
+                self.move_type == "out_refund"
+                and not has_returned_qty
+                and remaining_qty
+                and line.product_id.type != "service"
+            ):
                 remaining_qty = 0.0
                 for key in picking_dict:
                     picking_dict[key] = abs(picking_dict[key])
             if not float_is_zero(
                 remaining_qty,
                 precision_rounding=line.product_id.uom_id.rounding or 0.01,
             ):
-                self._process_section_note_lines_grouped(
-                    previous_section, previous_note, lines_dict
-                )
```

Comment:
Original commit don't remove this, but It is moved inside:
`elif not line.move_line_ids and not line.sale_line_ids:`
at line 110